packages/component/src/Composer.js

```diff
@@ -131,19 +132,21 @@ function createFocusContext({ mainFocusRef, sendBoxRef }) {
       const ref = where === 'sendBox' || where === 'sendBoxWithoutKeyboard' ? sendBoxRef : mainFocusRef;
       const { current } = ref || {};
 
-      if (current && where === 'sendBoxWithoutKeyboard') {
-        // To not activate the virtual keyboard while changing focus to an input, we will temporarily set it as read-only and flip it back.
-        // https://stackoverflow.com/questions/7610758/prevent-iphone-default-keyboard-when-focusing-an-input/7610923
-        const readOnly = current.getAttribute('readonly');
+      if (current) {
+        if (where === 'sendBoxWithoutKeyboard') {
+          // To not activate the virtual keyboard while changing focus to an input, we will temporarily set it as read-only and flip it back.
+          // https://stackoverflow.com/questions/7610758/prevent-iphone-default-keyboard-when-focusing-an-input/7610923
```

Comment:
I love this

Reply:
A hack that have been working for last 9 years. 😂